omnij-jsonrpc/src/main/java/foundation/omni/rpc/OmniClient.java

```diff
@@ -2,24 +2,17 @@
 
 import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.JavaType;
-import com.google.common.primitives.UnsignedBytes;
-import org.consensusj.bitcoin.rpc.BitcoinExtendedClient;
-import foundation.omni.json.pojo.OmniPropertyInfo;
-import org.bitcoinj.core.LegacyAddress;
-import org.bitcoinj.core.SegwitAddress;
-import org.consensusj.bitcoin.rx.jsonrpc.RxBitcoinClient;
-import org.consensusj.jsonrpc.JsonRpcException;
-import org.consensusj.bitcoin.rpc.RpcConfig;
-import foundation.omni.CurrencyID;
-import foundation.omni.Ecosystem;
-import foundation.omni.OmniValue;
-import foundation.omni.PropertyType;
+import foundation.omni.*;
```

Comment:
I generally try to avoid wildcard imports, but I'm suspecting that IntelliJ did this automatically.